spaceship/lib/spaceship/client.rb

```diff
@@ -630,6 +630,22 @@ def fetch_program_license_agreement_messages
       return all_messages
     end
 
+    # Get provider news from App Store Connect's "olympus" endpoint
+    def fetch_provider_news_messages
+      all_messages = []
+
+      messages_request = request(:get, "https://appstoreconnect.apple.com/olympus/v1/providerNews")
+      body = messages_request.body
+      if body
+        body = JSON.parse(body) if body.kind_of?(String)
+        body.map do |messages|
+          all_messages.push(messages["message"])
```

Comment:
I followed the approach of the fetch_program_licence_agreement_messages, and returned only the string messages. Returning other fields wouldn't make any difference, as we cannot use the subject, for example, for comparing the messages as we are not sure if the same subject can be used for multiple different messages. Also, we are not using the priority or the group as we are checking for and showing all of them. Let me know if you disagree.

Reply:
I'd say it's ok since we don't have a use case for those fields at the moment.